.craft.yml

```diff
@@ -3,6 +3,9 @@ github:
   repo: sentry-toolbar
 preReleaseCommand: bash bin/bump-version.sh
 targets:
+  - name: github
```

Comment:
This is the right kind of thing! https://github.com/getsentry/craft?tab=readme-ov-file#github-github